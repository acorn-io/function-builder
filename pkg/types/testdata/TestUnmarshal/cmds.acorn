targets: one: {
	from: "required"
	run:  "echo hi"
}

targets: two: {
	from: "required"
	run: [
		"one",
		"two",
		"three",
	]
}

targets: three: {
	from: "required"
	run: [
		"one",
		{
			command: "two"
		},
		"three",
	]
}
