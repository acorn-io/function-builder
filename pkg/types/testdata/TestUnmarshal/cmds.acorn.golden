{
  "targets": {
    "one": {
      "from": "required",
      "workdir": "/src",
      "run": [
        {
          "command": "echo hi"
        }
      ]
    },
    "three": {
      "from": "required",
      "workdir": "/src",
      "run": [
        {
          "command": "one"
        },
        {
          "command": "two"
        },
        {
          "command": "three"
        }
      ]
    },
    "two": {
      "from": "required",
      "workdir": "/src",
      "run": [
        {
          "command": "one"
        },
        {
          "command": "two"
        },
        {
          "command": "three"
        }
      ]
    }
  }
}